### DISCUSSION TITLE
error TS5109: Option 'moduleResolution' must be set to 'Node16' (or left unspecified) when option 'module' is set to 'Node16'.

### DISCUSSION THREAD

**Discussion by ghost** [sapphiredev/cli#256] [Q&A] [answered]
I can't use the sapphire cli to start an app with a starter template.

```ps1
PowerShell 7.3.6
PS D:\Desktop\Projects\sns> sapphire new
√ What's the name of your project? ... my-sapphire-bot
√ Choose a language for your project » TypeScript (Recommended)
√ Choose a template for your project » Starter template (Recommended)
√ What format do you want your config file to be in? » JSON
√ What package manager do you want to use? » npm
√ Do you want to create a git repository for this project? ... no
✔ Cloning the repository
✔ Setting up the project
✔ Installing dependencies using npm
Done!
PS D:\Desktop\Projects\sns> cd my-sapphire-bot
PS D:\Desktop\Projects\sns\my-sapphire-bot> npm run dev

> my-sapphire-bot@1.0.0 dev
> npm run build && npm run start


> my-sapphire-bot@1.0.0 build
> tsc

error TS5109: Option 'moduleResolution' must be set to 'Node16' (or left unspecified) when option 'module' is set to 'Node16'.


Found 1 error.

PS D:\Desktop\Projects\sns\my-sapphire-bot> node --version
v18.12.1
```

**Comment by favna** [comment, marked as answer]
This will be fixed with https://github.com/sapphiredev/utilities/pull/637 which will be released within the next 30 or so minutes. You'll then be able to update to @sapphire/ts-config v5 (upgrading guide will be linked in the breaking changes notes) and it should work again. Alternatively for now lock your typescript to 5.1 by setting it to `~5.1.x` in your package.json

**Reply by ghost** [reply]
Thank you. A temporary solution is to pin TypeScript to 5.1.6, which now works
I look forward to seeing the new version uploaded to npm!

**Reply by favna** [reply]
The examples have been updated. No update to the CLI is necessary. You can generate a new project and the problem will be fixed.
